shell/browser/native_window_mac.mm

```diff
@@ -549,7 +551,7 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
 }
 
 bool NativeWindowMac::IsVisible() {
-  bool occluded = [window_ occlusionState] == NSWindowOcclusionStateVisible;
+  bool occluded = [window_ occlusionState] & NSWindowOcclusionStateVisible;
```

Comment:
```suggestion
  bool occluded = [window_ occlusionState] && NSWindowOcclusionStateVisible;
```

Reply:
That's incorrect. `&` is the intended operator here.

Reply:
doh ok... my bad